pkg/cloudprovider/controller/nodecontroller_test.go

```diff
@@ -704,6 +729,12 @@ func TestSyncNodeStatusTransitionTime(t *testing.T) {
 									Reason:             "Node health check succeeded: kubelet /healthz endpoint returns ok",
 									LastTransitionTime: util.Date(2012, 1, 1, 0, 0, 0, 0, time.UTC),
 								},
+								{
+									Type:               api.NodeSchedulable,
+									Status:             api.ConditionFull,
```

Comment:
No, Check comment in L690, existing node is schedulable so it should be ConditionFull.